docs/running-workflows/what-is-workflow/index.md

```diff
@@ -1 +1,88 @@
 # What is a workflow?
+
+Workflows describe which computational steps were taken to run an analysis.
+
+## Simple workflows
+
+Let us assume that our analysis is run in two stages, firstly a data filtering stage and secondly a data plotting stage. A hypothetical example:
+
+```console
+$ python ./code/mycode.py \
+    < ./data/mydata.csv > ./workspace/mydata.tmp
+$ python ./code/mycode.py --plot myparameter=myvalue \
+    < ./workspace/mydata.tmp > ./results/myplot.png
+```
+
+Note how we call a given sequence of commands to produce our desired output plots. In order to capture this sequence of commands in a “runnable” or “actionable” manner, we can write a short shell script run.sh and make it parametrisable:
```

Comment:
```suggestion
Note how we call a given sequence of commands to produce our desired output plots. In order to capture this sequence of commands in a “runnable” or “actionable” manner, we can write a short shell script `run.sh` and make it parametrisable:
```

Reply:
...also, a cosmetic comment: perhaps ASCII quotes ("foo") instead of Unicode quotes (“foo”) for simplicity? I'm always using the former... but that might be just an old habit.

Reply:
Fixed `run.sh`.